Lib/test/test_cmd_line.py

```diff
@@ -902,6 +902,26 @@ def res2int(res):
         )
         self.assertEqual(res2int(res), (6000, 6000))
 
+    def test_cpu_count(self):
+        code = "import os; print(os.cpu_count())"
+        res = assert_python_ok('-X', 'cpu_count=4321', '-c', code)
+        self.assertEqual(res.out.strip().decode("utf-8"), '4321')
+
+    def test_process_cpu_count(self):
+        code = "import os; print(os.process_cpu_count())"
+        res = assert_python_ok('-X', 'cpu_count=4321', '-c', code)
+        self.assertEqual(res.out.strip().decode("utf-8"), '4321')
```

Comment:
Honestly, you can merged this test with test_cpu_count(): test the two functions at once:

```
code = "import os; print(os.cpu_count(), os.process_cpu_count())"
```